project/DependsOn.scala

```diff
@@ -0,0 +1,21 @@
+import sbt._
+
+object DependsOn {
+
+  def scalatest(scalaVersion: String) =
+    if (scalaVersion.startsWith("2.12")) "org.scalatest" %% "scalatest" % "3.0.1" % Test
+    else "org.scalatest" %% "scalatest" % "2.2.6" % Test
+
+  def akkaActor(scalaVersion: String) =
+    scalaVersion match {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Default case not specified.](https://www.codacy.com/app/pathikrit/better-files/file/4972466329/issues/source?bid=2860326&fileBranchId=3676352#l10)